### PR TITLE
Adds custom reads for UserInfo

### PR DESCRIPTION
**Pull request by kenoir**
Fix for the Google update to the userinfo response.

### CODE REVIEW

**Comment by kenoir** [comment]
@steppenwells boop.

File: pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/googleAuthModel.scala [hunk 1]
```diff
@@ -26,7 +26,10 @@ object Token {
     (JsPath \ "id_token").read[String]
   )(Token.apply _)
 
-  def fromJson(json:JsValue):Token = Json.fromJson[Token](json).get
+  def fromJson(json:JsValue):Token = {
+    println(json)
```
Comment (steppenwells) [inline pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/googleAuthModel.scala]: remove

**Comment by theefer** [comment]
You could use Boolean.toString

**Comment by theefer** [comment]
Change to kind optionality presumably intentional? Looks good otherwise. :+1:

**Comment by steppenwells** [comment]
Looks good :+1:

File: pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/googleAuthModel.scala [hunk 2]
```diff
@@ -26,7 +26,9 @@ object Token {
     (JsPath \ "id_token").read[String]
   )(Token.apply _)
 
-  def fromJson(json:JsValue):Token = Json.fromJson[Token](json).get
+  def fromJson(json:JsValue):Token = {
```
Comment (rtyley) [inline pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/service/googleAuthModel.scala]: Don't think anything really changes here does it? Can probably leave that change out.

**Comment by theefer** [comment]
Nice fix :+1:

**Pull request merged**